apps/parsley/package.json

```diff
@@ -112,28 +110,14 @@
     "@types/prompts": "2.4.4",
     "@types/react": "18.0.15",
     "@types/react-dom": "18.2.25",
-    "@typescript-eslint/eslint-plugin": "6.19.1",
-    "@typescript-eslint/parser": "5.62.0",
     "@vitejs/plugin-react": "3.0.1",
     "babel-plugin-import-graphql": "2.8.1",
     "cypress": "12.7.0",
     "cypress-real-events": "1.11.0",
     "env-cmd": "10.1.0",
     "eslint": "8.31.0",
-    "eslint-config-airbnb": "19.0.4",
-    "eslint-config-prettier": "9.1.0",
-    "eslint-plugin-cypress": "2.12.1",
-    "eslint-plugin-import": "2.28.1",
-    "eslint-plugin-jest": "27.2.0",
-    "eslint-plugin-jsdoc": "^46.2.4",
-    "eslint-plugin-jsx-a11y": "6.7.1",
-    "eslint-plugin-prettier": "5.1.3",
-    "eslint-plugin-react": "7.33.2",
-    "eslint-plugin-react-hooks": "4.6.0",
-    "eslint-plugin-sort-destructure-keys": "^1.5.0",
+    "eslint-config-evg": "*",
     "eslint-plugin-sort-keys-plus": "^1.3.1",
```

Comment:
Can file a ticket to install `eslint-plugin-sort-keys-plus` in Spruce so we can extract it out later?

Reply:
DEVPROD-6890